packages/e2e-tests/specs/performance/site-editor.test.js

```diff
@@ -75,112 +77,123 @@ describe( 'Site Editor Performance', () => {
 		}, html );
 		await saveDraft();
 
-		id = await page.evaluate( () =>
+		postId = await page.evaluate( () =>
 			new URL( document.location ).searchParams.get( 'post' )
 		);
 	} );
 
 	afterAll( async () => {
+		const resultsFilename = basename( __filename, '.js' ) + '.results.json';
+		writeFileSync(
+			join( __dirname, resultsFilename ),
+			JSON.stringify( results, null, 2 )
+		);
```

Comment:
is storing this file alongside the sourcecode the optimal choice here? I know I was confused when first storing results files as artifacts because of the directory structure. would we want to move these instead to our relatively-new `__test-results` directory?

we could consider some kind of `TEST_RESULTS_BASE_DIR` `ENV` to hold the base path if we wanted to avoid doing path-math everywhere: running in CI vs. locally in Docker vs. locally without Docker…

Reply:
Agreed, we should be able to pass the `__test-results` path as an env variable and save directly there. Will give it a try.

Reply:
I've drafted the refactor in https://github.com/WordPress/gutenberg/pull/48684, but I'm not sure how to handle the "locally without docker" situation. I think it still makes the most sense (as it's most convenient) to store alongside the sourcecode when running e.g. via `npm run test:performance -- site-editor`.

Let me know if that makes sense to you.

Reply:
the main thing about colocating with sourcecode is that it leaves its mess distributed around the project. at least when creating a project-root-level `__test-results` directory it's easy to wipe that out in one command.

see for example challenges with the distributed `build`, `build-styles`, `build-modules`, and `build-types` directories in each package directory.

I'll add further remarks in the linked PR.

thanks! ✋